demos/projects/ESPRESSIF/az-ca-recovery/README.md

```diff
@@ -50,10 +51,17 @@ You may also need to enable long path support for both Microsoft Windows and git
 - Windows: <https://docs.microsoft.com/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd#enable-long-paths-in-windows-10-version-1607-and-later>
 - Git: as Administrator run `git config --system core.longpaths true`
 
+## Prepare the TLS CA Trust Bundle in the NVS
+
+Run the sample called `az-nvs-cert-bundle` in the `/ESPRESSIF` directory to load the v1.0 trust bundle in your ESP device. This will purposely save an incomplete trust bundle in your devices's NVS, which will then be loaded for the IoT application. Once the CA validation fails in this sample, the device will then move into the recovery phase which fetches the new and complete certificate trust bundle.
```

Comment:
Instead of going this route, we should document that:
1. The first bundle is Baltimore only
2. Initially Hub is configured with Baltimore (default)
3. Customers will need to use https://learn.microsoft.com/en-us/azure/iot-hub/migrate-tls-certificate?tabs=portal to switch to Digicert
4. Devices will be unable to connect after the change (may take hours due to DNS replication)
5. Device will perform recovery, download the new TrustBundle and successfully connect to Azure IoT.

Reply:
I worry that complicates the process. This is supposed to show the flow if the certs are invalid and it does that. There's already a lot of steps to flash the device and then there's the whole flow to create the Azure Function, set up a separate DPS and IoT Hub, etc. I would like to keep it more concise to show how this might work.

Reply:
I take it back: I agree with you that it's likely too late to use my proposal for testing. 
I assume we will be required to update this method once the service starts deploying the new CAs.